pkg/controllers/workapplier/controller.go

```diff
@@ -71,17 +77,135 @@ const (
 	workFieldManagerName = "work-api-agent"
 )
 
+const (
+	workAgeToReconcile = 1 * time.Hour
```

Comment:
can this also be a parameter with a default?

Reply:
Ah, makes sense. Updated (only updated the workapplier/controller.go and not the work/apply_controller.go)